korman/properties/modifiers/sound.py

```diff
@@ -368,6 +372,13 @@ def _convert_sound(self, exporter, so, pClass, wavHeader, dataSize, channel=None
             sound.properties |= plSound.kPropLooping
         if self.incidental:
             sound.properties |= plSound.kPropIncidental
+        if self.local_only:
+            # Local only and random sounds hate each other, so we must check for both
+            for i in bpy.data.objects:
+                if i.plasma_modifiers.random_sound:
+                    raise ExportError("SoundEmitter '{}': cannot have local only sounds and a random sound mod.".format(self.id_data.name))
+                else:
+                    sound.properties |= plSound.kPropLocalOnly
```

Comment:
This will raise an error if any object in the entire blend has a random sound modifier, which is not what you're intending, I suspect. What problem are you noticing with local only on random sounds?

Reply:
I think it's only if the random sound mod is referenced in a responder. IIRC, it causes URU to crash...and the dump mentioned that plRandomSoundMod doesn't have a local only reference (or it's named something different). I'll have to roll back and double check that though as the details are hazy at the moment.

If you look at line 375 above that selection, it only checks for the random sound mod if the sound(s) have local only enabled. I've tried exporting Direbo with the check and it exports the random sound mod properly if there's no local only sounds and raises the error if there is one or more within the set.

Reply:
After reverting and trying again, I can't seem to reproduce the error (if there even was one related to this PR; might have gotten things mixed up), so I've reverted the local only/random sound check.